### ISSUE TITLE
正负样本对

### ISSUE THREAD

**Issue by 1439278026** [PKU-ML/adainf#2]
请问正负样本对是如何构建的，文章中fig1显示负样本是经过数据增强。

**Comment by jizhejz** [comment]
是的。本文遵循标准的对比学习范式，通过数据增强构造正负样本对，来自同一张图片的增强样本为正样本对，不同图片的增强样本为负样本对。

**Comment by jizhejz** [comment]
@1439278026 是的。本文遵循标准的对比学习范式，通过数据增强构造正负样本对，来自同一张图片的增强样本为正样本对，不同图片的增强样本为负样本对
